clusters/app.ci/prow/03_deployment/retester.yaml

```
@@ -1,3 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  labels:
+    app: retester
+  name: retester
+  namespace: prow-monitoring
+spec:
+  jobLabel: app
+  selector:
+    matchLabels:
+      app: retester
```

This has to be the same as the label on the pod.

```suggestion
      app: prow
      component: retester
```